.gitignore

```diff
@@ -266,6 +266,7 @@ FakesAssemblies/
 /tools/**/node_modules/
 /skills/**/node_modules/
 /samples/**/node_modules/
+/generators/**/node_modules/
```

Comment:
One of my PRs replaced all this with just `**/node_modules` so we can catch all.

Reply:
Okay I will remove this then?

Reply:
Wait should I remove all of the node_modules gitignores then and have your PR handle this?

Reply:
Okay I have pulled your change and reverted mine